tests/h/views/activity_test.py

```diff
@@ -108,6 +108,467 @@ def pyramid_request(self, factories, pyramid_request):
         return pyramid_request
 
 
+@pytest.mark.usefixtures('annotation_stats_service', 'group_service', 'routes', 'search')
+class TestOpenGroupSearchController(object):
```

Comment:
The test classes are named after the class or function they test, so naming this `TestOpenGroupSearchController` could make someone wonder where `OpenGroupSearchController` is in the code. There are also quite a lot of duplicated tests here which I don't see the need for.

As long as we are using a single view class for handling open groups and regular groups with just a few special cases for open groups, then I would suggest adding tests as new methods to the existing `TestGroupSearchController` class.